packages/larva/scripts/upgrade.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This script installs the latest version of Larva
+ *
+ * @see https://github.com/penske-media-corp/pmc-larva/blob/master/UPGRADING.md#how-to-upgrade-a-larva-project
+ */
+
+var shell = require( 'shelljs' );
+
+console.log( 'Upgrading Larva to the latest stable version' );
+shell.exec( 'npm install @penskemediacorp/larva@latest', { silent: false } );
+
+console.log( 'Building your project' );
+shell.exec( 'npm run prod' );
```

Comment:
Looks great! We should also have scripts to clean the build directories, for when you add the write-json and parser scripts (the Gulp and webpack build already do cleaning for assets).

Also, is `shelljs` something to update the other scripts to use? Or doesn't matter? I think most of them use this kind of CLI wrapper functionality in Node.

Reply:
Sorry, outdated code, bumped with a cleaner version. I'll continue to iterate on these.

I'll take a look to see if anything can be refactored with `shelljs`, and there may even be a native way to do what I'm using it for - but research seemed to indicate it's solid, and the syntax is clean.